README.md

```diff
@@ -322,10 +322,11 @@ Also, you can export a host function. The example is available [here][json-examp
 `fmt.Printf` can be used in plugins if you attach `os.Stdout` as below. See [the example][wasi-example] for more details.
 
 ```Go
-p, err := cat.NewFileCatPlugin(ctx, cat.FileCatPluginOption{
-	Stdout: os.Stdout, // Attach stdout so that the plugin can write outputs to stdout
-	Stderr: os.Stderr, // Attach stderr so that the plugin can write errors to stderr
-})
+mc := wazero.NewModuleConfig().
+    WithStdout(os.Stdout). // Attach stdout so that the plugin can write outputs to stdout
+    WithStderr(os.Stderr). // Attach stderr so that the plugin can write errors to stderr
+    WithFS(f)              // Loaded plugins can access only files that the host allows.
+p, err := cat.NewFileCatPlugin(ctx, cat.FileCatPluginModuleConfig(mc))
```

Comment:
I guess `FileCatPluginModuleConfig` here is because it is hard to tell if another similar file is already generated like `WazeroModuleConfig`? Feels the latter is more intuitive..

I wonder if there are other codegen tools who have found a way to generate without writing the a redundant name into the same package? or is that even an issue? cc @Pryz in case something similar comes up in buf

Reply:
> I guess `FileCatPluginModuleConfig` here is because it is hard to tell if another similar file is already generated like `WazeroModuleConfig`? Feels the latter is more intuitive..

Yes, we can generate and use this name, but in case we have more than one `proto` plugin file in the same package, it's not possible to use it. Otherwise, we need to create this structure inside the plugin code itself or or make an assumption, signle plugin inside the single unique package.

Reply:
Maybe you can find another end user for an opinion as I'm not one. Another way is a small runtime lib dep which has the options available. I don't know the norms accepted in codegen... can you look at who is using go-plugin on GitHub and see if any are doing significant enough work to ping on this change? This is a big change to how things work, and worth at least 2 end users total.

Reply:
Yes, sure. Will try to find alternative options to have optimal solution for multiple end users.

Reply:
@codefromthecrypt a new version with separate `WazeroModuleConfig` in predefined plugin `options` package. Please review.

Reply:
I think the options package name is good for starters. In the future, there may be an issue around any deps that may exist for code gen, but aren't runtime concerns. we can wait for someone to complain about that.